### ISSUE TITLE
Caseflow product navigation bar: Users cannot use the keyboard to tab to the Caseflow > Switch Product element

### ISSUE THREAD

**Issue by alisan16** [department-of-veterans-affairs/caseflow#12122]
Users who are using only keyboard access (no mouse) cannot get to the Caseflow > Switch Product button. The Switch Product element cannot be accessed with a keyboard. (New: 08/20/2019) I see it is listed as text verses a drop down box or a menu item.
